extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -38,7 +38,10 @@ const renderShoppingTagTemplate = (tagData) => (
         }
       ></span>
       <span class="amp-story-shopping-tag-pill-text">
-        {tagData['product-tag-text'] || '$' + tagData['product-price']}
+        {!tagData['product-tag-text'] && (
+          <span class="amp-story-shopping-tag-currency-symbol">$</span>
+        )}
+        {tagData['product-tag-text'] || tagData['product-price']}
```

Comment:
Can this logic be simplified as the following?
```
        {tagData['product-tag-text'] || (
          <span class="amp-story-shopping-tag-currency-symbol">$</span>
          tagData['product-price']
        )}
```

Reply:
It may be possible to simplify however the approach you present will present problems when rendering in react, it will render as [object HTMLSpanElement] instead of the $.

Reply:
@alanorozco Would like to hear your thoughts if you have any on this matter - this problem requires some more advanced knowledge of react.